### ISSUE TITLE
Darwin panic: PANIC 56: uncaught error (calling abort)

### ISSUE THREAD

**Issue by vielmetti** [deoxxa/don#7]
Gist at

https://gist.github.com/f3e185e7f2dc40364faf95959ff8a920

Freshly downloaded binary from bintray, invoked as
```
/don_dev_darwin-amd64 --public_url=http://localhost:5000 --addr=":5000"
```

**Comment by deoxxa** [comment]
Yep, I think I've got go.rice configured incorrectly. Investigating now!

**Comment by deoxxa** [comment]
If you start it inside the don source directory though, so the `build` directory is available to it, and if you do `make build/entry-client-bundle.js build/entry-server-bundle.js` beforehand, it should work. Hopefully this will be solved momentarily though.

**Comment by deoxxa** [comment]
Actually, that's incorrect - it turned out to be a bug in the client code. Poor error reporting from duktape (my fault) hid a bug in the UI JavaScript. My react static rendering in go is still verrrry experimental. This does remind me though - I should tighten up the client code with flowtype now that it's actually working.
